Cargo.toml

```diff
@@ -43,7 +43,7 @@ tendermint-rpc                   = { version = "0.40.0" }
 tendermint-testgen               = { version = "0.40.0" }
 
 # Namada dependencies
-namada_sdk              = "0.47.1"
+namada_sdk              = { git = "https://github.com/anoma/namada", branch = "grarco/no-duplicated-events"  }
```

Comment:
Just a note: Need to update Namada SDK to the published version after released Namada